index.js

```diff
@@ -66,7 +66,8 @@ module.exports = (function () {
 function createExitHarness(conf, wait) {
     var config = conf || {};
     var harness = createHarness({
-        autoclose: defined(config.autoclose, false)
+        autoclose: defined(config.autoclose, false),
+        noOnly: defined(process.env.NODE_TAPE_NO_ONLY_TEST, false)
```

Comment:
let's allow this to be passed in with `conf`
```suggestion
        noOnly: defined(conf.noOnly, defined(process.env.NODE_TAPE_NO_ONLY_TEST, false))
```

Reply:
updated